client/my-sites/upgrades/checkout-thank-you/site-redirect-details.jsx

```diff
@@ -10,29 +10,38 @@ import { getDomainManagementUrl } from './utils';
 import i18n from 'lib/mixins/i18n';
 import PurchaseDetail from './purchase-detail';
 
-const SiteRedirectDetails = ( { selectedSite } ) => {
+const SiteRedirectDetails = ( { selectedSite, domain } ) => {
 	return (
 		<div>
 			<PurchaseDetail
 				additionalClass="redirect-now-working"
-				title={ i18n.translate( 'Redirect now working' ) }
-				description={ i18n.translate( 'Visitors to your site will be redirected to your chosen target.' ) }
-				buttonText={ i18n.translate( 'Test Redirect' ) }
-				href={ selectedSite.URL }
+				title={ i18n.translate( 'Test the redirect' ) }
+				description={
+					i18n.translate(
+						'Visitors to your site will be automatically redirected to {{em}}%(url)s{{/em}}.',
+						{
+							args: { url: domain },
+							components: { em: <em /> }
+						}
+					)
+				}
+				buttonText={ i18n.translate( 'Try it now' ) }
+				href={ `http://${ domain }` }
```

Comment:
Is there a reason that you switched to `http://${ domain }` instead of whatever the selected site's `URL` property is? Are we sure that `selectedSite.URL` always uses this protocol?